src/Chapter3.hs

```diff
@@ -910,6 +989,18 @@ Implement instances of "Append" for the following types:
 class Append a where
     append :: a -> a -> a
 
+data Gold = Gold Int
+
+data List = List [String]
+
+instance Append Gold where
+  append :: Gold -> Gold -> Gold
+  append (Gold x) (Gold y) = Gold (x + y)
+
+instance Append List where
+  append :: List -> List -> List
+  append (List x) (List y) = List (x ++ y)
```

Comment:
In this task it wasn't required to implement a separate `List` data type as a wrapper around single list. So you could implement the `Append` instance directly for `[a]`. But if you want to have a wrapper around any list, you can create a polymorphic data type:
```haskell
newtype List a = List [a]
```